tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -457,6 +457,28 @@ def mirror_snapshot_schedule_remove(self, **kwargs):
         cmd1 += f" {kwargs.get('starttime')}" if kwargs.get("starttime") else ""
         self.exec_cmd(cmd=cmd1)
 
+    def verify_snapshot_schedule_remove(self, **kwargs):
+        """
+        This will verify snapshot schedule got removed successfuly or not
+        Args:
+        **kwargs:
+            poolname : if poolname is specified schedule will be removed at pool level else cluster level
+            imagename : if imagename is specified schedule will be removed per image if not will done globally
+        Returns:
+            0 on success
+        """
+        cmd1 = "rbd mirror snapshot schedule list"
+        if kwargs.get("poolname"):
+            cmd1 += f" --pool {kwargs.get('poolname')}"
+        if kwargs.get("imagename"):
+            cmd1 += f" --image {kwargs.get('imagename')}"
+        output = self.exec_cmd(output=True, cmd=cmd1, check_ec=False)
+        if not output:
+            log.info("snapshot schedule got removed successfully")
+            return 0
+        else:
+            raise Exception("Failed to remove snapshot schedule")
```

Comment:
@sunilangadi2 according to discussion with @VasishtaShastry please add 
log.error("Failed to remove snapshot schedule")
return 1

instead of raise Exception

Reply:
Thanks @manasagowri Done the changes,
Please review it.